src/card/basic/card.html

```diff
@@ -1,15 +1,16 @@
 <div ng-class="$ctrl.showTopBorder === 'true' ? 'card-pf card-pf-accented' : 'card-pf'">
   <div ng-if="$ctrl.showHeader()" ng-class="$ctrl.shouldShowTitlesSeparator ? 'card-pf-heading' : 'card-pf-heading-no-bottom'">
-    <div ng-if="$ctrl.showFilterInHeader()" ng-include="'card/basic/card-filter.html'"></div>
+    <div ng-if="$ctrl.showFilterInHeader() && !$ctrl.showSpinner" ng-include="'card/basic/card-filter.html'"></div>
     <h2 class="card-pf-title">{{$ctrl.headTitle}}</h2>
   </div>
+  <span ng-if="$ctrl.subTitle && !$ctrl.showSpinner" class="card-pf-subtitle">{{$ctrl.subTitle}}</span>
 
-  <span ng-if="$ctrl.subTitle" class="card-pf-subtitle">{{$ctrl.subTitle}}</span>
-
-  <div class="card-pf-body">
-    <div ng-transclude></div>
+  <div class="card-pf-body" ng-class="{'show-spinner': $ctrl.showSpinner}">
+    <label ng-if="!$ctrl.spinnerText" class="sr-only">Loading</label>
+    <div ng-if="$ctrl.showSpinner" class="spinner spinner-lg" aria-hidden="true"></div>
+    <div ng-class="{'hide-body': $ctrl.showSpinner}" ng-transclude></div>
```

Comment:
Shouldn't we be displaying $ctrl.spinnerText if it is set?

Reply:
Working on that now :)

Reply:
whoops did that backwards. Fixing.. :)